### ISSUE TITLE
Hyper-V VM doesn't work

### ISSUE THREAD

**Issue by papersaltserver** [minoca/os#57]
Hello, I want to test Minoca OS using Hyper-V 2016 server. But unfortunately, OS can't initialize properly, it can't finish boot sequence. All I can see is initial information. Any ideas, how can I force it to start?
![minocaos_hyperv_failed_to_boot](https://cloud.githubusercontent.com/assets/8975555/20339306/a65cfd88-abec-11e6-9ca3-65df786dc683.PNG)

**Comment by evangreen** [comment]
I'm currently looking into this. In the meantime, the latest images built from source should work on VMWare and VirtualBox. We hope to put out new binary images in the next few days, so if you don't feel like compiling, sit tight.

**Comment by evangreen** [comment]
I pushed change f79e6612a97da58c9e43daf4734f9c1cdc355c5c , which gets me to a prompt on Hyper-V. I realized though that we don't have NIC drivers for Hyper-V, so there's no network access. This change also came in slightly too late to make the 0.3 release. For now I think I'd still recommend VirtualBox or VMWare. If you don't mind using it without networking, then feel free to compile the latest master.
